src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -28,6 +31,7 @@ object SharedFileSystem {
     case "copy" => localizePathViaCopy _
     case unsupported => throw new UnsupportedOperationException(s"Localization strategy $unsupported is not recognized")
   })
+  val CallLocalizers = Stream(localizePathAlreadyLocalized _, localizePathViaSymbolicLink _, localizePathViaHardLink _, localizePathViaCopy _)
```

Comment:
Workflow inputs have configurable localization strategies, which we've defaulted to hard -> soft -> copy, and to which we prepend already-localized.  For calls it looks like we're hardcoding all the strategies, and the order is different from the default order of workflow inputs.  Is there a reason for this?